recipe/meta.yaml

```diff
@@ -21,6 +21,7 @@ requirements:
     - pkg-config
     - {{ stdlib('c') }}
     - {{ compiler('c') }}
+    - {{ stdlib("c") }}
```

Comment:
Is this necessary? Does order matter? Isn't it the same as the line 2 above?

Reply:
Yeah it is the same as the line above.  Order should not matter.